Makefile

```diff
@@ -128,3 +129,21 @@ gen-docs:
 		-w /helm-charts \
 		$(HELM_DOCS_IMAGE) \
 		helm-docs
+
+# Synchronize alerting rules in charts/victoria-metrics-k8s-stack/templates/rules
+sync-rules:
+	docker run --rm --name sync-rules \
```

Comment:
I'd suggest to remove `name` as it can produce errors in case previous container will remain in system due to some reason, docker will fail to create a new one since name already exists.
Also, it would be great to add `--user $(shell id -u):$(shell id -g)` to map current user to container and avoid issues with files being modified/created by root in linux.

Reply:
Fixed